minicluster/src/test/java/alluxio/multi/process/MultiProcessClusterTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.multi.process;
+
+import static org.junit.Assert.assertEquals;
+
+import alluxio.AlluxioURI;
+import alluxio.Constants;
+import alluxio.PropertyKey;
+import alluxio.client.file.FileInStream;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.FileSystemTestUtils;
+import alluxio.client.file.options.CreateFileOptions;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.Timeout;
+
+public final class MultiProcessClusterTest {
+  private boolean mSuccess;
+  private MultiProcessCluster mCluster;
+
+  @Rule
+  public Timeout mTimeout = Timeout.millis(Constants.MAX_TEST_DURATION_MS);
+
+  @Before
+  public void before() {
+    mSuccess = false;
+  }
+
+  @Test
+  public void simpleCluster() throws Exception {
+    mCluster = MultiProcessCluster.newBuilder()
+        .setClusterName("simpleCluster")
+        .setNumMasters(1)
+        .setNumWorkers(1)
+        .build();
+    try {
+      mCluster.start();
+      FileSystem fs = mCluster.getFileSystemClient();
+      createAndOpenFile(fs);
+      mSuccess = true;
+    } finally {
+      cleanup();
+    }
+  }
+
+  @Test
+  public void zookeeper() throws Exception {
+    mCluster = MultiProcessCluster.newBuilder()
+        .setClusterName("zookeeper")
+        .addProperty(PropertyKey.ZOOKEEPER_ENABLED, "true")
+        .setNumMasters(3)
+        .setNumWorkers(2)
+        .build();
+    boolean success = false;
```

Comment:
is this var ever used any where?

Reply:
removed, thanks - it was left over from a previous refactor